src/components/ConfigurableResource/ConfigurableResources.tsx

```diff
@@ -1,29 +1,28 @@
-import ChevronRightIcon from 'mdi-react/ChevronRightIcon'
-import Link from 'next/link'
+import { ResourceLink } from 'components'
 import { IConnection } from 'types/Connection'
+import { JWTSession } from 'types/JWTSession'
+
 interface IProps {
   connection: IConnection
+  jwt: string
+  token: JWTSession
 }
 
-const ConfigurableResources = ({ connection }: IProps) => {
-  const { configurable_resources, unified_api, service_id } = connection
+const ConfigurableResources = ({ connection, jwt, token }: IProps) => {
+  const { configurable_resources } = connection
+
   return (
     <div className="overflow-hidden bg-white border rounded-md">
       <ul className="divide-y divide-gray-200">
         {configurable_resources.map((resource: string, index: number) => {
           return (
-            <li key={`resource-${index}`}>
-              <Link href={`/integrations/${unified_api}/${service_id}/${resource}`}>
-                <a className="flex items-center justify-between px-4 py-3 text-gray-800 capitalize text-sm font-medium sm:px-6 hover:bg-gray-50 group">
-                  <span>{resource}</span>
-                  <ChevronRightIcon
-                    className="transition duration-150 ease-in-out transform group-hover:translate-x-1"
-                    color="currentColor"
-                    size={24}
-                  />
-                </a>
-              </Link>
-            </li>
+            <ResourceLink
```

Comment:
Small thing but I would move the list element up because it's not really part of the link
```
<li>
   <ResourceLink />
</li>
 ```